src/components/address-output/_macro.spec.js

```diff
@@ -5,94 +5,121 @@ import * as cheerio from 'cheerio';
 import axe from '../../tests/helpers/axe';
 import { renderComponent } from '../../tests/helpers/rendering';
 
-const EXAMPLE_ADDRESS_OUTPUT_FULL = {
-    unit: 'Unit 5',
-    organisation: 'Trescos',
-    line1: 'Abingdon Road',
-    line2: 'Goathill',
-    town: 'Barry',
-    postcode: 'AB12 6UH',
-};
-
-const EXAMPLE_ADDRESS_OUTPUT_NONE = {};
-
-describe('macro: address-output', () => {
-    it('passes jest-axe checks', async () => {
-        const $ = cheerio.load(renderComponent('address-output', EXAMPLE_ADDRESS_OUTPUT_FULL));
-
-        const results = await axe($.html());
-        expect(results).toHaveNoViolations();
+import { EXAMPLE_ADDRESS_OUTPUT_FULL, EXAMPLE_ADDRESS_OUTPUT_NONE } from './_test_examples';
+
+describe('FOR: address-output', () => {
+    describe('GIVEN: Params: default', () => {
+        describe('WHEN: All params are at default state', () => {
```

Comment:
These params aren't default ones. I think this needs to be reworded. This is whats being passed in here:
unit: 'Unit 5',
organisation: 'Trescos',
line1: 'Abingdon Road',
line2: 'Goathill',
town: 'Barry',
postcode: 'AB12 6UH'

Maybe something like "All address line params are provided"

Reply:
the jest-axe tests confused me as well - sometimes they are done with default, other times with just required and still others with all pararms. i feel like we should be keeping this consistent across testing docs

Reply:
Yeah this is something we need to investigate. It might be that it does differ based on component. Can you create a card to investigate this @balibirchlee?